packages/gatsby-source-drupal/src/gatsby-node.js

```diff
@@ -176,13 +177,20 @@ exports.onCreateDevServer = (
     bodyParser.text({
       type: `application/json`,
     }),
-    async (req, res) => {
-      // we are missing handling of node deletion
-      const nodeToUpdate = JSON.parse(JSON.parse(req.body)).data
-
-      await handleWebhookUpdate(
+    async (req, _res) => {
+      const requestBody = JSON.parse(JSON.parse(req.body))
+      const { data, secret, action, id } = requestBody
```

Comment:
Nice! It seems like this is backward compatible with previous version of drupal's gatsby module?

Reply:
it would be yes, however deleteNode is not accessible, that I believe would be a change that would have to happen in `onCreateDevServer`

Reply:
`deleteNode` is an action, so you should be able to use `actions.deleteNode`

Reply:
great, thank you